test/metabase/api/user_test.clj

```diff
@@ -60,34 +60,36 @@
 
 (deftest user-list-test
   (testing "GET /api/user"
-    (testing "Check that anyone can get a list of all active Users"
+    (testing "Check that admins can get a list of all active Users"
       (mt/with-non-admin-groups-no-root-collection-perms
-        (is (= [{:id          (mt/user->id :crowberto)
-                 :email       "crowberto@metabase.com"
-                 :first_name  "Crowberto"
-                 :last_name   "Corv"
-                 :common_name "Crowberto Corv"}
```

Comment:
Since the endpoint now only works for admins, the admins can see the entirety of a user entity, so it felt unnecessary to check for equivalence of the whole map. What I've done is added an assertion that the keys returned are what we expect for admins to be able to see, then just check that the correct users are returned by checking their emails, as I had done before.